S/Spot/build_tarballs.jl

```diff
@@ -0,0 +1,97 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "spot_julia"
```

Comment:
The name of the directory _must_ match this name.  Rename one of the two

Reply:
thanks, now at least the linux builds work :)